### ISSUE TITLE
link to docker images are dead

### ISSUE THREAD

**Issue by deep-42-thought** [NeutrinoToolkit/Neutrino#1]
Hi,
I'm trying to package neutrino for archlinux and noticed that the link to the docker images referenced in the README.md are dead.

On the other hand, it seems rather simple to compile neutrino via
```
mkdir build
cd build
cmake ..
make
make install
```
Why don't you just state the needed dependencies?
On archlinux, it turned out, I needed to install `fftw`, `gsl`, `qt5-multimedia`, `qt5-svg`, `qt5-tools` and solely for the compilation: `cmake`, `git` and `lsb-release`.

Cheers,
Erich

**Comment by iltommi** [comment]
Hi,
the two links I provide are 
debian docker: https://hub.docker.com/r/iltommi/debian-sid-neutrino/~/dockerfile/
and 
windows (cross compiled on fedora) https://hub.docker.com/r/iltommi/neutrino-docker-cross/~/dockerfile/
which are still alive (maybe I should update the docker image)

Yes it (rather) simple to compile but depending on the os (and distro on linux) packages names vary a lot. Moreover depending on the Qt version, things get more confusing (they keep changing stuff). 
On top of that if you want the python scripting shell plugin, you need to recompile pythonqt. Those docker images (and the travis.yml) are convenient if you want to recompile it by yourself and have all the features available.

Take them as a reminder (and yes, at the end of the dockerfile there are more or less the commands you give)

**Comment by deep-42-thought** [comment]
Oh, it seems to be a problem with *clicking* the link shown on github (my browser replaces `~` by `%7E`, which is displayed as `~` but docker does not give the proper content) - thanks for giving me the "real" link :-)
I'll have a look into pythonqt now, I currently don't understand, why there isn't / why I can't find a fitting packaged verion of it - It sounds like a general tool to me.

**Comment by iltommi** [comment]
There is quite a mess with pythonqt. The main point is that the builds-system has to change with each release of Qt and it's difficult to maintain since the cpp generator is not updated well (of course there are a lot of plans to ditch for clang, but this has not been done and I doubt we will see in the near future).

Another important point is that the creator (Florian) sticks to sourceforge (ok we can live with that) and using qmake to build, which is not a problem per se but someone wrote a cmakefile and Debian took it to package pythonqt. 

The cmakefile is quite a mess and it packages 2 different lis onto one (pythonqt and pytohnqt_qtall) resulting in a different implementation of pythonqt. 

Moreover the debian team did not update the package until very recently https://tracker.debian.org/pkg/pythonqt
switching back to qmake....

If things are like this, we might switch to qmake, but I did not succeed in cross building pytonqt for windows using mingw.

At the time we put pythonqt into neutrino, I found a cmakefile that did work for my purpose.
